run_params.yaml

```diff
@@ -619,6 +619,63 @@ simple_a2c_cartpole: &default_simple_a2c_params
     critic_network:
       hidden_size: 64
 
+selfplay_td3_driving: &default_selfplay_td3_params
+  implementation: selfplay_td3_training
+  config: &default_elfplay_td3_config
+    class_name: data_pb2.SelfPlayTD3TrainingRunConfig
+    environment:
+      specs:
+        implementation: driving/SimpleDriving-v0
+        num_players: 2
+      config:
+        seed: 12
+        render_width: 1000
+        framestack: 4
+    actor:
+      implementation: selfplay_td3
+      config:
+        num_input: 7
+        num_input_2: 2
+        num_action: 2
+        action_scale: [ 0.5,0.6 ]
+        action_bias: [ 1.0,0.0 ]
+        max_action: 1
+        alice_grid_shape: [75, 75, 2]
+        bob_grid_shape: [75, 75, 3]
+        actor_network:
+          hidden_size: 64
+        critic_network:
+          hidden_size: 64
+        model_kwargs:
+          start_timesteps: 5000
+          high_action: [ 0, -1 ]
+          low_action: [ 1, 1 ]
+          expl_noise: 0.1
+    rollout:
+      epoch_count: 1000
+      epoch_train_trial_count: 2
+      epoch_test_trial_count: 2
+      max_parallel_trials: 1
```

Comment:
can we increase this?

Reply:
Still having pybullet issues. So no, not until that's resolved.